server/src/middlewares/HandleCountyRequest.ts

```diff
@@ -0,0 +1,74 @@
+import { NextFunction, Request, Response } from 'express';
+
+import UserType from '../Models/User/UserType';
+import { DISTRICT_BY_COUNTY } from '../DBService/Counties/Query';
+import logger, { invalidDBResponseLog, launchingDBRequestLog, validDBResponseLog } from '../Logger/Logger';
+import { graphqlRequest, unauthorizedStatusCode } from '../GraphqlHTTPRequest';
+import { Severity } from '../Models/Logger/types';
+
+const handleCountyRequest = async (request: Request, response: Response, next: NextFunction) => {
+    const currentUser = response.locals.user;
+    const county = parseInt(request.body.county);
+
+    const countyMiddlewareLogger = logger.setup({
+        workflow: 'countyMiddleware',
+        investigation: currentUser.epidemiologynumber,
+    });
+
+    if (currentUser.userType === UserType.SUPER_ADMIN) {
```

Comment:
maybe export 'currentUser.userType' to const (line 33 also)

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1249/commits/51a1dbb29c72a734e562a38a45e089f0bea2498b)